src/utils/transactions/getOperationsDetails.ts

```diff
@@ -0,0 +1,48 @@
+import { InterpretedTransactionType, OperationType } from 'types';
+import { getVisibleOperations } from 'utils/transactions';
+
+type OperationDetailsPropsType = {
+  transaction: InterpretedTransactionType;
+  filterBy?: {
+    action?: OperationType['action'];
+    sender?: OperationType['sender'];
+    receiver?: OperationType['receiver'];
+  };
+};
+
+export const getOperationsDetails = ({
+  transaction,
+  filterBy
+}: OperationDetailsPropsType): OperationType[] => {
+  if (!transaction.operations) {
+    return [];
+  }
+
+  let operations: OperationType[] = getVisibleOperations(transaction);
```

Comment:
Doesn't  getVisibleOperations return a OperationType[]?  is this needed let operations: **OperationType[]**